client/my-sites/email/email-management/titan-management-nav/index.jsx

```diff
@@ -87,6 +99,59 @@ class TitanManagementNav extends React.Component {
 		);
 	};
 
+	renderMailboxSetupPrompt = () => {
+		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
+
+		const numberOfUnusedMailboxes =
+			getMaxTitanMailboxCount( domain ) - getConfiguredTitanMailboxCount( domain );
+
+		if ( numberOfUnusedMailboxes <= 0 ) {
+			return;
+		}
+
+		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
+		const controlPanelUrl = showExternalControlPanelLink
+			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } )
+			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } );
+
+		return (
+			<CompactCard className="titan-management-nav__mailbox-setup-prompt">
+				<span>
+					<Gridicon icon="info-outline" size={ 18 } />
+					<em>
+						{ translate(
+							'You have %(numberOfMailboxes)d unused mailbox',
+							'You have %(numberOfMailboxes)d unused mailboxes',
+							{
+								count: numberOfUnusedMailboxes,
+								args: {
+									numberOfMailboxes: numberOfUnusedMailboxes,
+								},
+								comment:
+									'This refers to the number of mailboxes purchased that have not been set up yet',
+							}
+						) }
+					</em>
+				</span>
+
+				{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
+				<Button
```

Comment:
Should we add tracking for this button too?

Reply:
Yes, I think we should. I'll work on that tomorrow.

Reply:
I forgot to add this in this PR -- I'll make sure I add it tomorrow.